scripts/take_day_flats.py

```diff
@@ -0,0 +1,49 @@
+from panoptes.utils import current_time
+from panoptes.utils import wait_for_events
+
+
+def take_pics(field,
+              observation,
+              cameras,
+              base_date=None,
+              num_exposures=1,
+              base_dir='/var/huntsman/images/temp'
+              take_flats=False):
+    """Takes pics with the cameras"""
+
+    # Set a base date for all pics and use that as directory name.
+    base_date = base_date or current_time(flatten=True)
+
+    for i in range(num_exposures):
+        print(f"Starting exposures {i:03d} of {num_exposures:03d}.")
+
+        exposure_events = []
+
+        # Share timestamp filename across all cameras.
+        timestamp = current_time(flatten=True)
+        for cam_name, camera in cameras.items():
+            new_filename = f'{base_dir}/{base_date}/{camera.uid}/{timestamp}.fits'
+
```

Comment:
Before doing `camera.take_observation`, it'll be good to move the filterwheel to the filter we want to use by using `camera.filterwheel.move_to(f'{filter_name}')`, so we need to add `filter_name` as an argument to the function `take_pics`. Currently, we have to manually move the filterwheel of each camera, thawhich could be done inside this function automatically for all the cameras.